docs/src/main/docs/health/01_health.adoc

```diff
@@ -241,3 +243,12 @@ Routing.builder()
     ]
 }
 ----
+
+=== Strict JSON Output
+
+The JSON responses shown above contain properties `"status"` and `"outcome"` with the same
+values. Helidon reports both of these to maintain backward compatibility with older
+versions of the MicroProfile Metrics library. This behavior can be disabled by setting
```

Comment:
Should this refer to the Health library, not Metrics?

Reply:
It didn't want that module to depend (even at runtime) on CDI. Originally, I put this new qualifier in `health-checks` but there were problems (can't depend on that module statically). Hence the creation of the new module.

Reply:
Yeap